### ISSUE TITLE
Java 11 support

### ISSUE THREAD

**Issue by regrog** [cincheo/jsweet-eclipse-plugin#28]
Hello

is this plugin available for the jdk 11?

Thanks

**Comment by renaudpawlak** [comment]
Nope. Sorry. Maintaining a plugin for Eclipse is too much work for the benefits. On our side, we set up the classpath accordingly in Eclipse and we transpile/build using:
- the JSweet Maven plugin for Maven projects, 
- or an Ant task (or even a main class) calling the JSweet's command line launcher class, for non-Maven projects.

**Comment by lgrignon** [comment]
Relates to https://github.com/cincheo/jsweet-eclipse-plugin/issues/29
